.appveyor.yml

```diff
@@ -1,13 +1,16 @@
 image: Visual Studio 2017
 configuration:
   - Release
+  - Debug
 platform:
   - x86
   - x64
-cache:
-  # Cache Nuget packages in "packages" folder. Break cache if "packages.config" is updated.
-  - packages -> **\packages.config
-before_build: nuget restore
+install:
+  - cmd: cd C:\Tools\vcpkg
+  - cmd: git pull
+  - cmd: .\bootstrap-vcpkg.bat
+  - cmd: vcpkg install gtest:x86-windows gtest:x64-windows
+  - cmd: cd %APPVEYOR_BUILD_FOLDER%
```

Comment:
Hmm, using this tool buries the setup/installation into an AppVeyor specific file. That doesn't make it easy for other developers to download the code from GitHub and get building. It would be very manual to get dependencies installed.

From what I understand, with NuGet, Visual Studio would know how to download and use the dependencies.

----

Small question, why the `git pull`?

Reply:
vcpkg comes preinstalled on appveyor. You could remove the Git pull and maybe some other lines. It just means that you wouldn't have the newest packages when downloading. This would of course improve your build times.

We would have to tell someone how to pull vcpkg and install gtest in the readme. A batch script could be developed since it all happens on the command line.

It takes about 3 console commands, so manual but I wouldn't say very manual.

Once a package is installed, it is available to all Visual Studio projects on the computer. So if you were using gtest in multiple projects, you would only download once.

You might be able to add installing gtest as a prebuild event in Visual Studio to automate. Not sure it is worth it either.

Brett